app/src/organisms/ModuleWizardFlows/CheckStackerInstall.tsx

```diff
@@ -0,0 +1,129 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { css } from 'styled-components'
+
+import {
+  COLORS,
+  Flex,
+  JUSTIFY_FLEX_END,
+  PrimaryButton,
+  RESPONSIVENESS,
+  SPACING,
+  TYPOGRAPHY,
+} from '@opentrons/components'
+import { FLEX_STACKER_MODULE_TYPE } from '@opentrons/shared-data'
+
+import { SmallButton } from '/app/atoms/buttons'
+import { SimpleWizardBody } from '/app/molecules/SimpleWizardBody'
+
+import type { AttachedModule } from '@opentrons/api-client'
+import type { DeckConfiguration } from '@opentrons/shared-data'
+import type { ModuleSetupWizardStepProps } from './types'
+
+interface CheckStackerInstallProps extends ModuleSetupWizardStepProps {
+  deckConfig: DeckConfiguration
+  attachedModules: AttachedModule[]
+  doorOpenStatus: boolean
+}
+
+const BUTTON_STYLE = css`
+    width: 100%;
+    justify-content: ${JUSTIFY_FLEX_END};
+    padding-right: ${SPACING.spacing32};
+    padding-bottom: ${SPACING.spacing32};
+
+    @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
+      justify-content: ${JUSTIFY_FLEX_END}};
+      padding-bottom: ${SPACING.spacing32};
+      padding-left: ${SPACING.spacing32};
+    }
+  `
+
+export function CheckStackerInstall(
+  props: CheckStackerInstallProps
+): JSX.Element {
+  const { proceed, isOnDevice, doorOpenStatus, attachedModules } = props
+  const { t, i18n } = useTranslation(['module_wizard_flows'])
```

Comment:
```suggestion
  const { t, i18n } = useTranslation(['module_wizard_flows', 'shared', 'branded'])
```
Then, when you're using copy from one of the second two files, preface the key with the file name like `shared:try_again`